packages/web-components/fast-foundation/CEMToMarkdown.mjs

```diff
@@ -216,7 +216,24 @@ function replaceJSDOCLinksWithMDLinks(text)
         {
             text = text.replace(jsdocLink, "[" + linkParts[1].trim() + "](" + linkParts[0].trim() + ")");
         }
-        startIndex = text.indexOf('{@link',endIndex);
+        startIndex = text.indexOf('{@link');
+    }
+    return text;
+}
+
+function removeCommentBlocks(text)
+{
+    let startIndex = text.indexOf('/*');
+    if(startIndex < 0) return text;
+
+    while(startIndex>=0)
+    {
```

Comment:
This style of formatting is only found in this file. Can you run Prettier for this file?

Reply:
It is a .mjs file in the package root so the normal prettier command wasn't seeing it. I figured out how to manually run prettier against it and have cleaned it up.